app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="container-b">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :email %>
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email' %>
+    </div>
+
+    <div class="field">
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control", placeholder: 'Username' %>
+    </div>
+    <div class="field">
+      <%= f.label :photo %>
+      <%= f.text_field :photo, class: "form-control", placeholder: 'Add photo link...' %>
+    </div>
+    <div class="field">
+      <%= f.label :bio  %>
+      <%= f.text_area :bio, class: "form-control", placeholder: 'User bio...' %>
+    </div>
+
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %>
+      <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password' %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %>
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: 'Confirm Password' %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up", class: 'signup-btn btn' %>
+      <% end %>
+
+      <%= render "devise/shared/links" %>
+    </div>
+</div>
```

Comment:
- [ ] Check the image for more info 
![image](https://user-images.githubusercontent.com/62705947/219147287-cafef684-1724-4b73-973d-9da870a1a850.png)

Reply:
- [ ] Check the image for more info 
![image](https://user-images.githubusercontent.com/62705947/219147649-d42dd07f-be08-4462-b7e2-51b6a9630454.png)